__tests__/models.test.js

```diff
@@ -68,6 +68,28 @@ describe("/api/reviews/:review_id", () => {
   });
 });
 
+describe("/api/reviews", () => {
+  it("should recieve a response of 200 and return all reviews ", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body }) => {
+        body.reviews.rows.forEach((review) => {
```

Comment:
remember to do a length check before a forEach, else you can get a false positive